packages/clerk-js/src/core/clerk.ts

```diff
@@ -1173,6 +1171,8 @@ export default class Clerk implements ClerkInterface {
         // set in updateClient
         this.updateEnvironment(environment);
 
+        await this.redirectFAPIInitiatedFlowIfSignedIn();
```

Comment:
🔧 i would expect that if a navigation is trigger to stop the execution of this function. Eg
```
if(await this.redirectFAPIInitiatedFlowIfSignedIn()) return;
```
For the above code to work we should return `true` from the `redirectFAPIInitiatedFlowIfSignedIn` when we trigger navigate.